.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "homebrew-formulae"]
 	path = homebrew-formulae
-	url = ./homebrew-formulae/
+	url = https://github.com/muter-mutation-testing/homebrew-formulae.git
```

Comment:
Why this change?

Reply:
git wasn't able to sync the submodule

Reply:
how did it ever work? Or was it always broken?

Reply:
🤷‍♂️